privacy/anonymizing-your-mac-address.md

```diff
@@ -34,10 +34,11 @@ wifi.scan-rand-mac-address=yes
 [connection]
 wifi.cloned-mac-address=stable
 ethernet.cloned-mac-address=stable
+connection.stable-id=${CONNECTION}/${BOOT}
 ~~~
 
-`stable` generates a random address that persists for each boot session.
-`random` generates a random address each time a link goes up.
+* `stable` in combination with `${CONNECTION}/${BOOT}` generates a random address that persists for each boot session.
```

Comment:
I think it isn't clear that address will be different after VM reboot. This is what will happen, right?
Maybe better write "... persists until reboot"?

Reply:
> I think it isn't clear that address will be different after VM reboot. This is what will happen, right?

Yes, that's what happens.

> Maybe better write "... persists until reboot"?

Agreed, that's clearer and easier to understand.